### ISSUE TITLE
Changes to local site build docs

### ISSUE THREAD

**Issue by dylanbmorgan** [maurergroup/MaurerGroupDocs#4]
I believe that in order to build the site locally, this [revision date localized plugin](https://pypi.org/project/mkdocs-git-revision-date-localized-plugin/) also needs to be installed, so maybe we should add that to the bottom of the homepage, or is this just as a result of my messed up local configuration of python?

**Comment by jamesgardner1421** [comment]
Yes that's correct, the [workflow file](https://github.com/maurergroup/MaurerGroupDocs/blob/main/.github/workflows/deploy.yml) will always contain the up to date build requirements if more extensions are added in the future so we could mention that too.
